### DISCUSSION TITLE
Flow solver Postprocessing

### DISCUSSION THREAD

**Discussion by tbellosta** [su2code/SU2#1927] [Q&A]
I'm implementing a convergence acceleration scheme that needs to modify the solution of the flow system (in the future maybe also the turb system) at each iteration of the pseudo-time integration. My original idea was to make all the magic happen in the solver's Postprocessing function, but it looks like the CSolver::Postprocessing method is not implemented for CFVMFlowSolverBase, nor for CEulerSolver. What is the SU2 style compliant way to postprocess the flow solution at each iteration?
Ideally, I would implement all common methods in CSolver and then call them in CFVMFlowSolverBase::Postprocessing and CTurbSolver::Postprocessing but I can only do it in the latter.
Should I re-overload the Postproceessing function for the CEulerSolver (or the CFVMFlowSolverBase) or is there a better way?

Thank you

**Comment by pcarruscag** [comment]
Hi, this should be done in one of the integration classes (multigrid integration for the flow solver, even if mg is not used)
